WooCommerce/Classes/ViewRelated/Coupons/Add and Edit Coupons/AddEditCoupon.swift

```diff
@@ -99,6 +118,25 @@ struct AddEditCoupon: View {
                             ListHeaderView(text: Localization.headerCouponDetails.uppercased(), alignment: .left)
 
                             Group {
+                                TitleAndValueRow(title: Localization.discountType,
+                                                 value: viewModel.discountTypeValue,
+                                                 selectionStyle: .disclosure, action: {
+                                    if idiom == .pad {
+                                        showingDiscountType.toggle()
+                                    } else {
+                                        discountTypeHandler(viewProperties)
+                                    }
```

Comment:
nit: I haven't worked with SwiftUI for a while and tried to remember the iOS 14 issue here 😅 could you briefly explain the issue? I don't see an issue with bottom sheet in the SwiftUI workarounds doc P91TBi-7F9-p2, could we add it here for future reference? When we drop iOS 14 support, we can update all the workarounds from iOS 14 in the doc.

Pretty minor but just wanted to mention: on iPad, the bottom sheet looks a bit tall in split view (iOS 15.5 simulator):

<img src="https://user-images.githubusercontent.com/1945542/170224624-65d409ed-edab-42b4-a04b-900fbdff55d8.png" width="300" />

Reply:
The issue with the bottom sheet was discussed here: p1653302908311579-slack-C02TD63FZA8.

So we want to be able to present a bottom sheet that covers only half of the screen, and this is not supported natively on iOS 14 yet, so I had to do this workaround. I'll include it on the P2 page when I finish this PR - thanks for reminding me!

I can't seem to repro the split view case on iPad, could you please include more information on how you got this error?

Reply:
Thanks for the link to the discussion! It's a shame many SwiftUI APIs are only added in recent iOS versions for developers to come up with workarounds (the drawback of adopting a new framework before it's fully featured 😞 ). I don't think this is an error and it's most likely a minor use case, it's probably the popover falling back to action sheet when the horizontal size class changes to compact. Totally fine to ship it with this, I think we had similar issues where the share sheet is full height using SwiftUI without a good solution. Just adding a note to migrate to use the iOS 15+ `adaptiveSheetPresentationController` bottom sheet should be good 👍 

I just tested in Xcode 13.3.1 (was using Xcode 13.4 earlier) iPad Air gen 4, and here's a recording:

https://user-images.githubusercontent.com/1945542/170404944-525b38a5-0e68-4816-8931-30c5f49d362d.mov

Reply:
I've added a comment in 072e8a8455bdfb18a124a0b9b70ece415a0a1a76 to remove the workaround when we drop iOS 14, and also update the SwiftUI workaround doc with a new section for this.